dbusmock/templates/bluez5.py

```diff
@@ -33,10 +37,20 @@
 ADAPTER_IFACE = 'org.bluez.Adapter1'
 MEDIA_IFACE = 'org.bluez.Media1'
 NETWORK_SERVER_IFACE = 'org.bluez.Network1'
+LEDADVERTISING_MNGR_IFACE = 'org.bluez.GattManager1'
 DEVICE_IFACE = 'org.bluez.Device1'
+GATT_MNGR_IFACE = 'org.bluez.GattManager1'
+GATT_SRVC_IFACE = 'org.bluez.GattService1'
+GATT_CHRC_IFACE = 'org.bluez.GattCharacteristic1'
+GATT_DSCR_IFACE = 'org.bluez.GattDescriptor1'
+
+# here = Path(__file__).parent
+# device_json = Path.joinpath('device_db.json')
+# with device_json.open() as input_db:
+#     microbit_data = json.load(input_db)
 
 
-def load(mock, _parameters):
+def load(mock, parameters):
```

Comment:
Is there any reason to rename that variable? Seems unused.

Reply:
Right, it's a sign that it's an unused formal parameter. Please keep.